internal/service/mapper/owner.go

```diff
@@ -36,6 +37,21 @@ func (s *Impl) GetOwner(ctx context.Context, ownerAlias string) (openapi.OwnerDt
 	fullPath := "owners/" + ownerAlias + "/owner.info.yaml"
 	err := GetT[openapi.OwnerDto](ctx, s, &result, fullPath)
 
+	if nil == err {
+		users, groups := util.SplitUsersAndGroups(result.Promoters)
```

Comment:
Actual only the promoters will be checked that they exists in bitbucket. What is with the users in owner.Groups, should they also be checked?